packages/spright-components/src/chat/input/styles.ts

```diff
@@ -0,0 +1,63 @@
+import { css } from '@ni/fast-element';
+import {
+    applicationBackgroundColor,
+    bodyFont,
+    bodyFontColor,
+    borderWidth,
+    controlLabelFontColor,
+    elevation2BoxShadow,
+    mediumPadding,
+    popupBorderColor
+} from '@ni/nimble-components/dist/esm/theme-provider/design-tokens';
+import { display } from '../../utilities/style/display';
+
+export const styles = css`
+    ${display('flex')}
+
+    :host {
+        width: 100%;
+        height: auto;
+    }
+
+    .container {
+        display: grid;
+        grid-template-rows: auto 1fr;
+        grid-template-columns: 1fr auto;
+        width: 100%;
+        height: 100%;
+
+        background-color: ${applicationBackgroundColor};
+        border: ${borderWidth} solid ${popupBorderColor};
+        box-shadow: ${elevation2BoxShadow};
+    }
+
+    textarea {
+        grid-row: 1;
+        grid-column: 1 / 3;
+
+        font: ${bodyFont};
+        color: ${bodyFontColor};
+        background-color: transparent;
+
+        width: 100%;
+        resize: none;
+        height: auto;
+        max-height: calc(6lh + 2 * ${mediumPadding});
+        field-sizing: content;
```

Comment:
As mentioned in the spec this isn't yet supported in Firefox and Safari but that's OK for our current use cases.